core/src/test/java/org/apache/any23/extractor/yaml/YAMLExtractorTest.java

```diff
@@ -93,4 +92,17 @@ public void nullTest()
         RepositoryResult<Statement> docs = getStatements(null, null, RDF.NIL);
         Assert.assertTrue(Iterations.asList(docs).size() == 2);
     }
+
+    /**
+     * Comma separated values are parsed as well.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void csvTest()
+            throws Exception {
+        assertExtract("/org/apache/any23/extractor/csv/test-comma.csv");
```

Comment:
That is false positive. It is just proof that for text-based formats additional recognising should be done from `meta`. I will delete that.